boards/snps/nsim/arc_classic/nsim_nsim_em.yaml

```diff
@@ -8,6 +8,8 @@ toolchain:
   - zephyr
   - cross-compile
   - arcmwdt
+supported:
+  - mpu
```

Comment:
Hmm, I'm wondering why this the only SNPS platform we add mpu tag to?
There are a lot of snps platforms which have mpu but now they will be filtered out in mpu-related tests.